### ISSUE TITLE
Crashes with William Wythers' Overhauled Overworld

### ISSUE THREAD

**Issue by siniel52** [Slexom/earth2java#141]
Hello, I noticed that Earth2Java and William Wythers' Overhauled Overworld not compatible. If both of mods active, world crashes in few mins after loading. Can you check it please? Thank you!
[crash-2024-07-21_13.09.31-fml.txt](https://github.com/user-attachments/files/16325777/crash-2024-07-21_13.09.31-fml.txt)
[latest.log](https://github.com/user-attachments/files/16325778/latest.log)
![image](https://github.com/user-attachments/assets/f014ab2e-d990-45ac-b45e-5ef609c0f9f9)

**Comment by Slexom** [comment]
E2J require [Cloth Config](https://www.curseforge.com/minecraft/mc-mods/cloth-config) to work.. Have you downloaded the mod manually? The launcher should have downloaded the dependencies too...
Unless the other mod doesn't embed its own version of cloth config but I don't think(hope) so...

**Comment by siniel52** [comment]
> E2J require [Cloth Config](https://www.curseforge.com/minecraft/mc-mods/cloth-config) to work.. Have you downloaded the mod manually? The launcher should have downloaded the dependencies too... Unless the other mod doesn't embed its own version of cloth config but I don't think(hope) so...

Yes I have it! Just accidentally deleted Cloth Config with all other mods to check maybe it's my problem
![image](https://github.com/user-attachments/assets/b284cfd7-53fb-4182-bfdd-9d077822a6b0)
![image](https://github.com/user-attachments/assets/037ce8d4-3e0d-4f6b-903e-19035a379e67)
![image](https://github.com/user-attachments/assets/1bb76a46-a1f3-48fd-8d36-842f3a8f021e)
![image](https://github.com/user-attachments/assets/51b1c573-9d6f-4842-a807-c723cc9d2e54)

So all dependencies downloaded, but still crashes.. thx for quick respons!

**Comment by siniel52** [comment]
[crash-2024-07-21_13.27.06-client.txt](https://github.com/user-attachments/files/16325805/crash-2024-07-21_13.27.06-client.txt)
Crash after ~5 min in new world, all my mods enabled

**Comment by siniel52** [comment]
I think all works good except 'fancy chicken egg'

**Comment by siniel52** [comment]
I checked all eggs. Game crashes after these two:
Cluckshroom
Fancy chicken
So WWOO had nothing to do with it
